README.md

```diff
@@ -66,7 +66,11 @@ $ chcon -R system_u:object_r:admin_home_t:s0 docker-elk/
 
 If you're using Docker for Windows, ensure the "Shared Drives" feature is enabled for the `C:` drive (Docker for Windows > Settings > Shared Drives). See [Configuring Docker for Windows Shared Drives](https://blogs.msdn.microsoft.com/stevelasker/2016/06/14/configuring-docker-for-windows-volumes/) (MSDN Blog).
 
-## Usage
+## Getting started
+
+### Version
+
+Change the ELK stack version in the _.env_ file.
```

Comment:
I'd rather move that to the customization section. The easiest would be to revert this: https://github.com/deviantony/docker-elk/commit/8f5b242d6a51615899ae920cc420b6393853ebcd